client/src/styles/leafletComponents/controlFrame.less

```diff
@@ -42,9 +43,13 @@
     background: #3F3A30;
     text-align: center;
     margin-top: 10px;
+    box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+
   }
 
   .c-control-frame__results {
     .s-container;
+    box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+
   }
 
```

Comment:
Yuck!  So a couple things here
1) This is not very DRY.  If you instead add it to `.s-container` which they all reference then you will get the same effect but only have to write it once.  You may need to add `.s-container` to the currently empty `.c-control-frame__search`
2) There is a LESS mixin for box-shadow that would be good to use.  http://lesselements.com/